cmd/buildkitd/main.go

```diff
@@ -256,6 +273,102 @@ func serveGRPC(c *cli.Context, server *grpc.Server, addrs []string, errCh chan e
 	return nil
 }
 
+func defaultConfigPath() string {
+	if system.RunningInUserNS() {
+		return appdefaults.UserConfigPath()
+	}
+	return appdefaults.ConfigPath
+}
+
+func defaultConf() (config.Config, *toml.MetaData) {
+	cfg, md, err := config.LoadFile(defaultConfigPath())
```

Comment:
what about `buildkitd --config myotherconfig.toml`  ?

Reply:
My bad

Reply:
this is just the default. `--config` will be loaded in a different line